signer/src/context/signer_state.rs

```diff
@@ -155,6 +155,21 @@ pub struct RollingWithdrawalLimits {
     pub cap: Amount,
 }
 
+impl RollingWithdrawalLimits {
+    /// Create a new one where the caps imply no withdrawals are allowed.
+    pub fn zero() -> Self {
+        RollingWithdrawalLimits { blocks: 0, cap: Amount::ZERO }
+    }
+
+    /// Create a new one where the caps imply all withdrawals are allowed.
+    pub fn unlimited() -> Self {
+        RollingWithdrawalLimits {
+            blocks: 0,
+            cap: Amount::MAX_MONEY,
```

Comment:
mmm we could theoretically exceed `Amount::MAX_MONEY` as when we sum withdrawals we could double count in case of withdrawal->deposit->withdrawal of the "same" amount. Maybe we should use u64::max?

Reply:
Oh right, great point. Will update.

Reply:
Maybe we could also add a test for MAX_MONEY + 1 and unlimited, to ensure we don't forget about it and change it back to amount

Reply:
Good call, done in https://github.com/stacks-network/sbtc/pull/1515/commits/b4975ebbd093def8ebd5febc73a8f18d0ce24a6a.